File: Nível 2/1009 - Salário com Bônus/main.lua
username = io.read()
salarioFixo = tonumber(io.read())
totalVendas = tonumber(io.read())

quantiaVendas = totalVendas * 0.15
salarioFinal = quantiaVendas + salarioFixo

print(string.format("TOTAL = R$ %.2f", salarioFinal))

File: Nível 2/1009 - Salário com Bônus/main.py
username = input("Nome")
salarioFixo = float(input("Qual o salario fixo: "))
totalVendas = float(input("Quanto de total de vendas: "))

quantiaVendas = totalVendas * 0.15
salarioFinal = quantiaVendas + salarioFixo

print("TOTAL = R$ %.2f" % salarioFinal)